### PR TITLE
BETA: Register kuroigg.is-a.dev

### PR DESCRIPTION
**Pull request by kuroigg**
Added `kuroigg.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

File: domains/johndoegg.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "kuroigg",
+        "email": "mrlangdon197@gmail.com"
+    },
+    "record": {
+        "CNAME": "johndoegg.github.io"
```
Comment (CuteDog5695) [inline domains/johndoegg.json:7]: 404 page

File: domains/kuroigg.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "kuroigg",
+        "email": "mrlangdon197@gmail.com"
+    },
+    "record": {
+        "CNAME": "kuroigg.github.io"
```
Comment (CuteDog5695) [inline domains/kuroigg.json:7]: please add more stuff to your website, not just "Hello World" text

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/kuroigg.json (hunk 2 above)
Comment (kuroigg) [reply domains/kuroigg.json:7]: About that, its currently work in progress, and its just a placeholder.

**Review by iostpa — APPROVED** [review]
seems fine to me now

**Pull request merged**